test.sh

```diff
@@ -56,6 +56,31 @@ function test_metrics() {
 
 # ==================================================================================
 
+function test_entrypoint() {
+  local name=$1
+  local entrypoint=$2
+
+  local container_id=$(docker run --name ${name}-test -d \
+                         -e LANG=en_US.UTF-8 -e PARAMETER_THAT_MAY_NEED_ESCAPING="&'\"|< é\\(" ${name} \
+                         ${entrypoint} --commandLineArgValueThatMayNeedEscaping="&'\"|< é\\(" --killDelay=1 --exitCode=0)
+
+  # sleep is required because after docker run returns, the container is up but our server may not quite be yet
```

Comment:
remove this left over comment (as it does not `sleep` here, contrary to orig test)